src/applications/hca/config/chapters/householdInformation/annualIncome.js

```diff
@@ -0,0 +1,118 @@
+import { merge, set, get } from 'lodash/fp';
+import fullSchemaHca from 'vets-json-schema/dist/10-10EZ-schema.json';
+import currencyUI from 'platform/forms-system/src/js/definitions/currency';
+
+import { incomeDescription } from '../../../helpers';
+import { validateCurrency } from '../../../validation';
+import {
+  dependentIncomeUiSchema,
+  createDependentIncomeSchema,
+} from '../../../definitions/dependent';
+
+const dependentIncomeSchema = createDependentIncomeSchema(fullSchemaHca);
+
+const isVeteranMarried = formData =>
+  formData.maritalStatus?.toLowerCase() === 'married';
+
+const isVeteranSeparated = formData =>
+  formData.maritalStatus?.toLowerCase() === 'separated';
+
+const hasVeteranBeenMarried = formData =>
+  (formData.maritalStatus && isVeteranMarried(formData)) ||
+  isVeteranSeparated(formData);
+
+const {
+  dependents,
+  spouseGrossIncome,
+  spouseNetIncome,
+  spouseOtherIncome,
+  veteranGrossIncome,
+  veteranNetIncome,
+  veteranOtherIncome,
+} = fullSchemaHca.properties;
+
+export default {
+  uiSchema: {
+    'ui:title': 'Annual income',
+    'ui:description': incomeDescription,
+    veteranGrossIncome: set(
+      'ui:validations',
+      [validateCurrency],
+      currencyUI('Veteran\u2019s gross annual income from employment'),
+    ),
+    veteranNetIncome: set(
+      'ui:validations',
+      [validateCurrency],
+      currencyUI(
+        'Veteran\u2019s net income from your farm, ranch, property or business',
+      ),
+    ),
+    veteranOtherIncome: set(
+      'ui:validations',
+      [validateCurrency],
+      currencyUI('Veteran\u2019s other income amount'),
+    ),
+    'view:spouseIncome': {
+      'ui:title': 'Spouse\u2019s income',
+      'ui:options': {
+        hideIf: formData =>
+          !formData.maritalStatus ||
+          (!isVeteranMarried(formData) && !isVeteranSeparated(formData)),
```

Comment:
This took me a minute, but now it makes sense 😅

Reply:
@Mottie do you have any suggestions on making it easier to digest?

Reply:
Naw, the `hideIf` function always throws me off... it's fine as it is

Reply:
Yeah I feel that my brain hurts every time I switch between `depends` and `hideIf`